styleguide/source/assets/js/form-items.js

```diff
@@ -30,6 +29,27 @@
           $('.textarea').keyup(function() {
             count_remaining_character();
           });
+
+          // Range Field
+          var legend = $('.ama__range-field__legend');
+          var handle = $( "#currentValue" );
```

Comment:
Use single quotes and remove spaces. See above code.

Reply:
Yeah ur right